_docs/treeview/treeview.md

```diff
@@ -0,0 +1,131 @@
+# TreeView Component
+
+**TreeView** is a component that creates a hierarchical display of data. Example uses of TreeViews include a corporate hierarchy, a directory structure, a documentation center, and so on.
+
+## Elements
+
+![TreeView example](./assets/treeview.png)
```

Comment:
This was a placeholder and a proper image has now been supplied by the components team.